utilities/transactions/transaction_lock_mgr.h

```diff
@@ -28,6 +28,25 @@ struct LockInfo;
 struct LockMap;
 struct LockMapStripe;
 
+struct DeadlockInfoBuffer {
+ private:
+  std::vector<std::vector<DeadlockInfo>> paths_buffer;
+  uint32_t buffer_idx;
```

Comment:
lets stick to convention and use `variable_` with the trailing underscore for all the private member variables